docs/app/components/navigation-bar-search/navigation-bar-search.component.ts

```diff
@@ -160,4 +182,24 @@ export class NavigationBarSearchComponent {
     isDuplicate(item: ISearchResult) {
         return this.results.filter((result: ISearchResult) => result.link.title === item.link.title).length > 1;
     }
+
+    /**
+     * Add an item to the bounded history list.
+     */
+    private addToHistory(item: ISearchResult) {
+
+        // Remove the item if it's already there.
+        const historyIndex = this.history.indexOf(item);
```

Comment:
The old site stored the last 5 results in local storage so they persisted across page refreshes. I realise this was required on the old site as it wasnt a single page application, however do you think this feature should be carried across?